File: Python/Program/Product.py
class Product:
    def __init__(self, id=0, name="", brand="", price=""):
        self.id = id
        self.name = name
        self.brand = brand
        self.price = price

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_brand(self):
        return self.brand

    def set_brand(self, brand):
        self.brand = brand

    def get_price(self):
        return self.price

    def set_price(self, price):
        self.price = price

File: CPP/Program/Product.cpp
#include <iostream>
#include <string>

using namespace std;

class Product
{
    private: 
        int id;
        string name;
        string brand;
        string price;

    public:
        Product()
        {
            this->id = 0;
            this->name = "";
            this->brand = "";
            this->price = "";
        }

        Product(int id, string name, string brand, string price)
        {
            this->id = id;
            this->name = name;
            this->brand = brand;
            this->price = price;
        }

        int get_id() const
        {
            return this->id;
        }

        void set_id(int id)
        {
            this->id = id;
        }

        string get_name() const
        {
            return this->name;
        }

        void set_name(string name)
        {
            this->name = name;
        }

        string get_brand() const
        {
            return this->brand;
        }

        void set_brand(string brand)
        {
            this->brand = brand;
        }

        string get_price() const
        {
            return this->price;
        }

        void set_price(string price)
        {
            this->price = price;
        }

        ~Product()
        {

        }

};